src/dstack/_internal/core/backends/digitalocean_base/compute.py

```diff
@@ -155,9 +158,9 @@ def _get_image_for_instance(self, instance_offer: InstanceOfferWithAvailability)
             return "ubuntu-24-04-x64"
 
         gpu_count = len(instance_offer.instance.resources.gpus)
-        gpu_name = instance_offer.instance.resources.gpus[0].name
+        gpu_vendor = instance_offer.instance.resources.gpus[0].vendor
 
-        if gpu_name == "MI300X":
+        if gpu_vendor == "amd":
```

Comment:
(nit) Use the enum variant instead.

```suggestion
        if gpu_vendor == gpuhunt.AcceleratorVendor.AMD:
```

This ensures that the line will keep working properly even if `AcceleratorVendor` values change